### ISSUE TITLE
Collection query: pagination broken - can only include one of "before" or "after"

### ISSUE THREAD

**Issue by cspecter** [supabase/pg_graphql#199]
**Describe the bug**
We have an infinite list component with a pull-to-refresh function. When the list reaches the end it will run a `fetchMore` methos with the `pageInfo.endCursor`, when you pull to refresh it will send `pageInfo.startCursor`.

Currently you can not include both a `before` and `after` key in the query.

**To Reproduce**
ToDo Query

```graphql
query TasksCollectionQuery(
  $completed: Boolean!
  $first: Int
  $last: Int
  $before: Cursor
  $after: Cursor
) {
  tasksCollection(
    filter: { completed: { eq: $completed } }
    first: $first
    last: $last
    before: $before
    after: $after
  ) {
    edges {
      node {
        id
        title
        completed
      }
    }
    pageInfo {
      endCursor
      hasNextPage
      hasPreviousPage
      startCursor
    }
  }
}
```

Query variables

```json
{
  "after": "ABCD1234..."
}
```

Response

```json
{
  "data": null,
  "errors": [
    {
      "message": "only one of \"before\" and \"after\" may be provided"
    }
  ]
}
```

**Expected behavior**
API should expect both a before and after cursor in one query and return items to be paginated.

**Versions:**
Public version

**Comment by olirice** [comment]
duplicate of #161

I'll add more context on that thread
